src/main/java/ch/admin/bar/siardsuite/presenter/archive/ArchiveRecentConnectionsDialogPresenter.java

```diff
@@ -0,0 +1,158 @@
+package ch.admin.bar.siardsuite.presenter.archive;
+
+import ch.admin.bar.siardsuite.component.CloseDialogButton;
+import ch.admin.bar.siardsuite.util.fxml.FXMLLoadHelper;
+import ch.admin.bar.siardsuite.util.fxml.LoadedFxml;
+import ch.admin.bar.siardsuite.util.i18n.DisplayableText;
+import ch.admin.bar.siardsuite.util.i18n.keys.I18nKey;
+import ch.admin.bar.siardsuite.util.preferences.DbConnection;
+import ch.admin.bar.siardsuite.util.preferences.UserPreferences;
+import ch.admin.bar.siardsuite.view.DialogCloser;
+import io.github.palexdev.materialfx.controls.MFXButton;
+import javafx.fxml.FXML;
+import javafx.geometry.Insets;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import lombok.val;
+
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+public class ArchiveRecentConnectionsDialogPresenter {
+
+    private static final I18nKey ARCHIVE_TITLE = I18nKey.of("archiveDbDialog.title");
+    private static final I18nKey ARCHIVE_TEXT = I18nKey.of("archiveDbDialog.text");
+
+    private static final I18nKey UPLOAD_TITLE = I18nKey.of("archiveDbDialog.title");
```

Comment:
This file should not be in archive-folder, because now it is used for both workflows - archive and upload. 

same for `fxml` files